ticdc/ticdc-changefeed-config.md

```diff
@@ -111,16 +127,33 @@ dispatchers = [
 # The protocol configuration item specifies the protocol format of the messages sent to the downstream.
 # When the downstream is Kafka, the protocol can only be canal-json or avro.
 # When the downstream is a storage service, the protocol can only be canal-json or csv.
+# Note: ignore this field when the downstream type is the DB.
```

Comment:
Does “DB” mean “TiDB”？

Reply:
DB means MySQL compatible database, it is an internal data type

Reply:
have changed to Database